### PR TITLE
HW1 Commit

### PR DESCRIPTION
**Pull request by eesalt**
Added HW1.py file for the first assignment. Reading in Iris data, generating pandas df, gathering general stats with numpy, practicing with plotly, and doing some sklearn intro into classifiers and pipelines

### CODE REVIEW

**Comment by eesalt** [comment]
Please review my HW1 assignment

File: HW1.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    print(np.min(df[{"Sepal Length", "Sepal Width", "Petal Length", "Petal Width"}]))
+    print("Maxes")
+    print(np.max(df[{"Sepal Length", "Sepal Width", "Petal Length", "Petal Width"}]))
+    print("25th percentiles:")
+    print(df.quantile(0.25, numeric_only=True))
+    print("50th percentiles:")
+    print(df.quantile(0.50, numeric_only=True))
+    print("75th percentiles:")
+    print(df.quantile(0.75, numeric_only=True))
+
```
Comment (matherdy) [inline HW1.py:32]: Instead of plotting multiple scatter plots, maybe try using Plotly's scatter matrix

File: HW1.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        color="Class",
+    )
+    scatter.show()
+
+    # parallel categories
+    parallel = px.parallel_categories(
+        df_norm, color="value", color_continuous_scale=px.colors.sequential.Purp
+    )
+    parallel.show()
+
```
Comment (matherdy) [inline HW1.py:80]: Instead of printing the results maybe try printing the accuracy of the predict

File: HW1.py (hunk 1 above)
Comment (eesalt) [reply HW1.py:32]: I started out with the basic scatters but I've got a scatter matrix a bit further down.

**Pull request merged**